src/backend/distributed/operations/shard_cleaner.c

```diff
@@ -572,15 +585,17 @@ FinalizeOperationNeedingCleanupOnSuccess()
 
 	if (list_length(currentOperationRecordList) > 0)
 	{
-		ereport(LOG, (errmsg("Removed %d orphaned shards out of %d",
-							 removedShardCountOnComplete, list_length(
-								 currentOperationRecordList))));
+		ereport(LOG, (errmsg(
+						  "Processed %d orphaned shards out of %d as part of operation finalize ",
```

Comment:
similar feedback. I also liked the previous "Removed" wording a little more than "Processed".

Reply:
Hey @marcocitus , I thought the term "Removed" as part of inline cleaning in operation was "concerning" as you mentioned here : https://github.com/citusdata/citus/issues/6312. Why are we going back to it?

Reply:
Fixed as per offline discussion.